site/docs/foundations/typography.mdx

```diff
@@ -3,161 +3,36 @@ title: Typography
-
-<Image
-  src="/img/foundations/typography-colors.svg"
-  alt="Example of type color options"
-/>
-
-| Token                                 | Figma                        | Light mode       | Dark mode       |
-| ------------------------------------- | ---------------------------- | ---------------- | --------------- |
-| `--salt-content-primary-foreground`   | Content/Primary/Foreground   | `color-gray-900` | `color-white`   |
-| `--salt-content-secondary-foreground` | Content/Secondary/Foreground | `color-gray-200` | `color-gray-70` |
-
-### Primary foreground
-
-Use the primary text color by default. It should comprise the majority of text within any composition. It provides the greatest level of contrast between foreground and background.
-
-### Secondary foreground
-
-You should use secondary text sparingly for supporting text, or for contrast between labels and data.
-
-<DesignCards>
-
-<DesignCard
-      src="/img/foundations/typography-secondary-foreground-1.svg"
-      alt="Example of a form field"
->
-
-Form field labels use secondary text to give greater prominence to the entered value.
-
-</DesignCard>
-
-<DesignCard
-      src="/img/foundations/typography-secondary-foreground-2.svg"
-      alt="Example of a group of label value pairings"
->
-
-Use secondary text for labels to ensure data has greater prominence without the need for differing weights or font sizes.
-
-</DesignCard>
-
-</DesignCards>
-
-<ExampleContainer type="positive">
-  <Image
-    src="/img/foundations/typography-heading-emphasized.svg"
-    alt="Example showing heading emphasized with font weight"
-  />
-  Use primary and secondary text colors and/or weight variants to create visual
-  hierarchy.
-</ExampleContainer>
-
-<ExampleContainer type="negative">
-<Image
-  src="/img/foundations/typography-color-heading.svg"
-  alt="Example showing heading colored incorrectly"
-/>
-
-Don’t use colors, other than the primary and secondary options, to emphasize headings, labels, alerts or blocks of text.
+The legacy Salt theme uses the open-source, sans-serif typeface **Open Sans**. It’s the sole typeface for all text styles, with the exception of code which uses the mono space typeface, PT Mono.
 
-</ExampleContainer>
+## Font weight
 
-All text within the design system meets the Web Content Accessibility Guidelines (WCAG) 2.0 level AA requirements to ensure legibility.
+Weight refers to the relative thickness of a font’s stroke. Each type style includes three weights which enable you to create visual hierarchy within the same text size.
 
-WCAG 2.0 requires that the contrast ratio between text and background is at least 4.5:1 for normal-sized text and 3:1 for larger text.
+Please note that in the Salt design system, 'Strong' indicates bolder weight than the default, whereas 'Small' represents a thinner weight, which may differ from traditional terminology. 
```

Comment:
This is standard web technology term, e.g. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong

Reply:
Changed this to specify audience - this is not common terminology for designers.

Reply:
What part isn't common?  strong text?

Reply:
We don't use small like it usually is used natively (smaller text) we instead use it to drop the weight. Which might be what the original sentence meant.